proto/regen/group/v1alpha1/genesis.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+package regen.group.v1alpha1;
+
+option go_package = "github.com/regen-network/regen-ledger/x/group";
+
+import "gogoproto/gogo.proto";
+
+// Params defines the set of configurable parameters.
+message Params {
+    option (gogoproto.equal) = true;
+    option (gogoproto.goproto_stringer) = false;
+
+    // max_comment_length defines the maximum comment length of the group module genesis params.
+    uint32 max_comment_length = 1 [(gogoproto.moretags) = "yaml:\"max_comment_length\""];
+}
+
+// GenesisState defines the group module's genesis state.
+message GenesisState {
+    option (gogoproto.equal) = true;
+    option (gogoproto.goproto_stringer) = false;
+
+    // params are the genesis state Params.
+    Params params = 1 [(gogoproto.nullable)=false];
+}
```

Comment:
Don't we also need to include groups, group accounts, proposals, etc?

Reply:
I'm fine if that's a separate PR but let's just make sure we have an issue open.

Reply:
indeed, let's make it part of a separate PR, I added a new issue: https://github.com/regen-network/regen-ledger/issues/214
it also seems like there was already some WIP on that: https://github.com/regen-network/cosmos-modules/pull/60/files